picard/ui/options/dialog.py

```diff
@@ -263,3 +371,62 @@ def _show_dialog(self, msg, function):
         message_box.setStandardButtons(QtWidgets.QMessageBox.Yes | QtWidgets.QMessageBox.No)
         if message_box.exec_() == QtWidgets.QMessageBox.Yes:
             function()
+
+
+class AttachedProfilesDialog(PicardDialog):
+    NAME = 'attachedprofiles'
+    TITLE = N_('Attached Profiles')
+
+    def __init__(self, parent=None, option_group=None):
+        super().__init__(parent=parent)
+        self.option_group = option_group
+        self.ui = Ui_AttachedProfilesDialog()
+        self.ui.setupUi(self)
+        self.ui.buttonBox.addButton(StandardButton(StandardButton.CLOSE), QtWidgets.QDialogButtonBox.RejectRole)
+        self.ui.buttonBox.rejected.connect(self.close_window)
+
+        self.populate_table()
+
+        self.ui.buttonBox.setFocus()
+        self.setModal(True)
+
+    def populate_table(self):
+        model = QtGui.QStandardItemModel()
+        model.setColumnCount(2)
+        header_names = [_("Option Setting"), _("Attached Profiles")]
+        model.setHorizontalHeaderLabels(header_names)
+
+        config = get_config()
+        profiles = config.profiles[SettingConfigSection.PROFILES_KEY]
+        settings = config.profiles[SettingConfigSection.SETTINGS_KEY]
+
+        group = UserProfileGroups.SETTINGS_GROUPS[self.option_group]
+        group_title = group["title"]
+        group_options = group["settings"]
+
+        window_title = _("Profiles Attached to Options in %s Section") % group_title
+        self.setWindowTitle(window_title)
+
+        for name, title in group_options:
+            item = QtGui.QStandardItem(_(title))
+            item.setEditable(False)
+            row = [item]
+            attached = []
+            for profile in profiles:
+                if name in settings[profile["id"]]:
+                    attached.append("{0}{1}".format(profile["title"], _(" [Enabled]") if profile["enabled"] else "",))
+            attached_profiles = "\n".join(attached) if attached else _("None")
+            item = QtGui.QStandardItem(attached_profiles)
+            item.setEditable(False)
+            row.append(item)
```

Comment:
It took me a while to notice `item` here is different from `item` above, perhaps just use 2 different variable names.

Reply:
Changed to use different names as suggested.  Makes it a lot less confusing.  Good catch!